components/TextFields/src/ColorThemer/MDCTextFieldColorThemer.m

```diff
@@ -49,14 +49,6 @@ + (void)applyColorScheme:(id<MDCColorScheme>)colorScheme
   if ([textInputControllerClass respondsToSelector:@selector(setActiveColorDefault:)]) {
     [textInputControllerClass setActiveColorDefault:colorScheme.primaryColor];
   }
-
-  if ([textInputControllerClass
```

Comment:
Does the same change need to be made for `floatingPlaceholderNormalColor`? It seems like the default would not change instances that are explicitly themed.

Reply:
Very good catch!

Reply:
Of course, this whole thing makes me wonder if I should be using the secondary color at all.........